sources/packages/api/src/services/institution/institution.service.ts

```diff
@@ -157,13 +151,25 @@ export class InstitutionService extends RecordDataModelService<Institution> {
     return await this.save(institution);
   }
 
+  async syncInstitution(userInfo: UserInfo): Promise<void> {
+    const account = await this.bceidService.getAccountDetails(
+      userInfo.idp_user_name,
+    );
+    const institutionEntity = await this.getInstitute(userInfo);
+    institutionEntity.guid = account.institution.guid;
+    institutionEntity.legalOperatingName = account.institution.legalName;
+    await this.save(institutionEntity);
+    const user = await this.userService.getUser(userInfo.userName);
```

Comment:
If the user is already loaded in the institutionEntity, please consider changing the user object inside institutionEntity and executing only one save operation.

Reply:
The egar loading working, so FIXED